### ISSUE TITLE
Translations with interpolations break on rails 7.1+

### ISSUE THREAD

**Issue by robbevp** [svenfuchs/i18n-active_record#146]
While upgrading to rails 7.1.2, I bumped into an issue with translations that have interpolations.
The following record will throw an error, since it tries to parse the value as YAML
```ruby
t = I18n::Backend::ActiveRecord::Translation.create(
  key: 'count',
  locale: 'en',
  value: '%{count}',
  is_proc: false,
  interpolations: "---\n- count\n"
)
t.reload.value
```
This error and backtrace:
```
ruby/gems/3.2.0/gems/psych-5.1.1.1/lib/psych/parser.rb:62:in `_native_parse': (<unknown>): could not find expected directive name while scanning a directive at line 1 column 1 (Psych::SyntaxError)
ruby/gems/3.2.0/gems/psych-5.1.1.1/lib/psych/parser.rb:62:in `_native_parse'
ruby/gems/3.2.0/gems/psych-5.1.1.1/lib/psych/parser.rb:62:in `parse'
ruby/gems/3.2.0/gems/psych-5.1.1.1/lib/psych.rb:455:in `parse_stream'
ruby/gems/3.2.0/gems/psych-5.1.1.1/lib/psych.rb:399:in `parse'
ruby/gems/3.2.0/gems/psych-5.1.1.1/lib/psych.rb:323:in `safe_load'
ruby/gems/3.2.0/gems/activerecord-7.1.1/lib/active_record/coders/yaml_column.rb:37:in `load'
ruby/gems/3.2.0/gems/activerecord-7.1.1/lib/active_record/coders/column_serializer.rb:37:in `load'
ruby/gems/3.2.0/gems/activerecord-7.1.1/lib/active_record/type/serialized.rb:22:in `deserialize'
ruby/gems/3.2.0/gems/activemodel-7.1.1/lib/active_model/attribute_set/builder.rb:52:in `block in fetch_value'
ruby/gems/3.2.0/gems/activemodel-7.1.1/lib/active_model/attribute_set/builder.rb:46:in `fetch'
ruby/gems/3.2.0/gems/activemodel-7.1.1/lib/active_model/attribute_set/builder.rb:46:in `fetch_value'
ruby/gems/3.2.0/gems/activerecord-7.1.1/lib/active_record/attribute_methods/read.rb:33:in `read_attribute'
ruby/gems/3.2.0/gems/i18n-active_record-1.3.0/lib/i18n/backend/active_record/translation.rb:106:in `value'
```

I'm running Rails 7.1.2 and i18n-active_record 1.3.0

This error is caused by 
https://github.com/svenfuchs/i18n-active_record/blob/fce95f2b344c8cec4a69934aa0db3f7842eb6574/lib/i18n/backend/active_record/translation.rb#L56-L61
I'm not sure why the value gets interpreted as YAML instead of just text, but I might be missing something.

Happy to provide a PR with a failing test!

**Comment by timfjord** [comment]
A failing test would be awesome.

**Comment by robbevp** [comment]
Hi @timfjord 

Apologies, I was a bit too quick to conclude that this was an issue in i18n-active_record. 
While trying to recreate this issue, I found that it was a deprecation in rails. I've opened an issue there, and will close this one.

For context: We have a table translations that contains the key, and a value per language `value_en`, `value_de`, .... to make it easier to manage translations. We then use a SQL view to generate one row per key/language.
These values in our first model weren't serialized. Rails was still able to parse these in 7.0, but isn't in 7.1, resulting in this error.
